### PR TITLE
housekeeping

### PR DESCRIPTION
**Pull request by retokromer**
- "modern" shebang
- uniformed indenting to 4
- $(...) preferred over \`...\`
- = preferred over == inside [[...]]
- ${VAR} preferred over $VAR

### CODE REVIEW

File: dev-installer [hunk 1]
```diff
@@ -21,126 +21,121 @@
 # @author Joseph Perry <joseph@artefactual.com>
 # @version svn: $Id$
 
-cd `dirname $0`
+cd $(dirname "${0}")
 
 part="create the archivematica user"
 echo -n "\"Would you like to ${part}?\" (y/N) "
 read a
-if [[ $a == "Y" || $a == "y" ]]; then
```
Comment (Hwesta) [inline dev-installer:29]: Why is `=` preferred over `==` inside `[[...]]`? I couldn't find an explanation via Google.

**Review by Hwesta — COMMENTED** [review]
This looks good, thanks for the contribution! Can you sign a [contributor's agreement](https://wiki.archivematica.org/images/2/25/Contributor_agreement.txt) and email it to agreement@artefactual.com ? Thank you!

By the way, `dev-installer` and `dev-helper` are considered deprecated - we recommend using [Ansible and Vagrant](https://wiki.archivematica.org/Getting_started#Installation) for development or a [packge install](https://www.archivematica.org/docs/latest/admin-manual/installation/installation/). We're also interested if you have feedback on making the documentation clearer around this.

File: dev-installer (hunk 1 above)
Comment (retokromer) [inline dev-installer]: I’ll send you the technical reference as soon as back home.

Comment (retokromer) [reply dev-installer:29]: In `[[ ... ]]` there is no reason to use `==`. However, if you prefer, I can undo.

**Comment by Hwesta** [comment]
Cherry-picked to qa/1.x as 379ad89212ba0f7d5ea6968ec13c659c77e7fb84 and 2f9b602abd0ce4be042892ec084dc070c2aaeecf  Thanks for the cleanup!

**Pull request closed without merging**